azure-nextgen-ts-webapp-privateendpoint-vnet-injection/index.ts

```diff
@@ -0,0 +1,158 @@
+// Copyright 2016-2021, Pulumi Corporation.  All rights reserved.
+
+// pvt DNS zone and virtual network link are only available on this version
+import * as pvtnetwork from "@pulumi/azure-nextgen/network/v20180901";
+import * as network from "@pulumi/azure-nextgen/network/v20200401";
+import * as resources from "@pulumi/azure-nextgen/resources/latest";
+import * as web from "@pulumi/azure-nextgen/web/v20190801";
+import * as pulumi from "@pulumi/pulumi";
+import * as random from "@pulumi/random";
+
+const config = new pulumi.Config();
+
+// setup a resource group
+const resourceGroupName = config.get("resourceGroupName") || "webapp-pvtendpoint-vnet-injection";
+const location = config.get("location") || "westus2";
+const resourceGroup = new resources.ResourceGroup("resourceGroup", {
+    resourceGroupName: resourceGroupName,
+    location: location,
+});
+
+const serverfarm = new web.AppServicePlan("serverfarm", {
+    kind: "app",
+    location: location,
+    name: "appServerFarm",
+    resourceGroupName: resourceGroup.name,
+    sku: {
+        capacity: 1,
+        family: "P1v2",
+        name: "P1v2",
+        size: "P1v2",
+        tier: "PremiumV2",
+    },
+});
+
+// To get a random suffix
+const rand = new random.RandomString("random", {
+    length: 5,
+    special: false,
+});
+
+// Setup backend app
+const backendName = pulumi.interpolate`backend${rand.result}`;
+const backendApp = new web.WebApp("backendApp", {
+    kind: "app",
+    location: location,
+    name: backendName,
+    resourceGroupName: resourceGroup.name,
+    serverFarmId: serverfarm.id,
+    siteConfig: {
+        ftpsState: web.FtpsState.AllAllowed,
```

Comment:
Do you know if we need this setting?

Reply:
No don't think so, came from the arm template.